src/auth/components/AvailablePortals.tsx

```diff
@@ -112,56 +114,55 @@ export const AvailablePortals: React.FC<{
   const history = useHistory()
   const [setPortal] = useSetPortalMutation()
 
+  const selectPortalHandler = (portal: string) => {
+    if (currentPortal === portal) {
+      history.push('/')
+      return
+    }
+
+    if (
+      window.confirm(
+        `Are you sure you want to go to the ${
+          displayNames[portal as Portal] ?? portal
+        } portal?`,
+      )
+    ) {
```

Comment:
I prefer an early return 🤓 
```suggestion
    if (
      !window.confirm(
        `Are you sure you want to go to the ${
          displayNames[portal as Portal] ?? portal
        } portal?`,
      )
    ) {
      return
    }
```